setup.py

```diff
@@ -1,139 +1,132 @@
-    "sentencepiece~=0.1.96": ["install"],
-    "protobuf~=3.20.0": ["install"],
-    "tqdm~=4.64.0": ["install"],
-    "transformers~=4.17.0": ["install"],
+    "transformers~=4.17.0": ["install", "gpu"],
+    "sentencepiece~=0.1.96": ["install", "gpu"],
     "ujson~=5.1.0": ["install"],
-    "transformers~=4.17.0": ["install"],
-    "tqdm~=4.64.0": ["install"],
-    "frozendict": ["install"],
-    "nlp": ["install"],
-    "sentencepiece~=0.1.96": ["install"],
-    "protobuf~=3.20.0": ["install"],
-    "nltk~=3.6":["install"],
-    "tabulate~=0.8.9":["install"],
-    "rouge_score":["install"], 
+    "tqdm~=4.64.0": ["install", "gpu"],
+    "frozendict": ["install", "gpu"],
+    "nlp": ["install", "gpu"],
+    "protobuf~=3.20.0": ["install", "gpu"],
+    "tabulate~=0.8.9": ["install", "gpu"],
+    "rouge_score": ["install", "gpu"],
     "myst-parser~=0.17.2": ["docs"],
     "pydata-sphinx-theme~=0.9.0": ["docs"],
     "sphinx~=4.4.0": ["docs"],
     "sphinx_design~=0.2.0": ["docs"],
-    "recommonmark~=0.7.1":["docs"]
+    "recommonmark~=0.7.1": ["docs"],
 }
 
-extras_names = ["docs", "dev", "install", "notebooks", "tests"]
+extras_names = ["docs", "dev", "install", "notebooks", "tests", "gpu"]
 extras = {extra_name: [] for extra_name in extras_names}
-for package_name, package_required_by in _deps.items():
-    if not package_required_by:
-        raise ValueError(f"Package '{package_name}' has no associated extras it is required for. "
-                         f"Choose one or more of: {extras_names}")
+for dep_package_name, dep_package_required_by in _deps.items():
+    if not dep_package_required_by:
+        raise ValueError(
+            f"Package '{dep_package_name}' has no associated extras it is required for. "
+            f"Choose one or more of: {extras_names}"
+        )
 
-    for extra_name in package_required_by:
+    for extra_name in dep_package_required_by:
         try:
-            extras[extra_name].append(package_name)
+            extras[extra_name].append(dep_package_name)
         except KeyError as ex:
             raise KeyError(f"Extras name '{extra_name}' not in {extras_names}") from ex
 extras["all"] = list(_deps)
 
 install_requires = extras["install"]
 
-python_version = sys.version_info.major,sys.version_info.minor
-
-"""
-if python_version == (3,7):
-    install_requires.append("torch-scatter @ https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_scatter-2.0.9-cp37-cp37m-linux_x86_64.whl")
-elif python_version == (3,8):
-    install_requires.append("torch-scatter @ https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_scatter-2.0.9-cp38-cp38-linux_x86_64.whl")
-elif python_version == (3,9):
-    install_requires.append("torch-scatter @ https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_scatter-2.0.9-cp39-cp39-linux_x86_64.whl")
-"""
-
 setup(
-    name="primeqa",
+    name=package_name,
+    url="https://github.com/primeqa/primeqa",
     version=version,
-    author=", ".join(authors),
+    author="PrimeQA Team",
```

Comment:
@avisil Please check the author list change from individual names to generic PrimeQA Team